WordPress/Classes/Extensions/Math.swift

```diff
@@ -22,14 +22,13 @@ extension Int {
 }
 
 extension Comparable {
-    /**
-     Clamps self between a minimum and maximum value
-     
-     - returns: the method returns
-        - min if self < min
-        - max if self > max
-        - otherwise it returns self
-     */
+    /// Clamps self between a minimum and maximum value
+    ///
+    /// - Returns: the method returns
```

Comment:
Capitalized is fine IMHO and makes the text closer to human writing and easier to read.